easybuild/easyconfigs/p/pyobjcryst/pyobjcryst-2.1.0.post2-intel-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'pyobjcryst'
+version = '2.1.0.post2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/diffpy/pyobjcryst'
+description = "Python bindings to ObjCryst++, the Object-Oriented Crystallographic Library."
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['2ee797cf4d0ba90a7d144110b0f3295ee7fa73462c648297100bfd5eb467bb6d']
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),  # for numpy
+    ('Boost.Python', '1.72.0'),
+    ('libobjcryst', '2017.2.2'),
```

Comment:
@zarybnicky This should be bumped to 2017.2.3 too (and please remove `libobjcryst-2017.2.2-intel-2020a.eb `, you can use `eb --update-pr 11321 :libobjcryst-2017.2.2-intel-2020a.eb` for that)

Reply:
Really? That's what I did, it even showed `easybuild/easyconfigs/l/libobjcryst/libobjcryst-2017.2.2_sconscript_local.patch | 20 --------------------` :man_facepalming:

Reply:
Even the commit in the PR itself displays it as a file rename, I'm confused...
I'll squash, I guess, that'll fix it.